lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -33,7 +33,9 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
     valueEl.textContent = Util.formatDisplayValue(audit.result.displayValue);
 
     const descriptionEl = this.dom.find('.lh-metric__description', tmpl);
-    descriptionEl.appendChild(this.dom.convertMarkdownLinkSnippets(audit.result.description));
+    if (audit.result.description) {
```

Comment:
a little confused by removing the `?` from `description` but making this conditional... what's up with that?

Reply:
good catch. it's assumed to be there now.